docs/index.md

```diff
@@ -98,7 +98,7 @@ or as a development dependency for your project:
 $ npm install --save-dev mocha
 ```
 
-> As of v6.0.0, Mocha requires Node.js v6.0.0 or newer.
+> As of v6.0.0, Mocha requires Node.js v8.0.0 or newer.
```

Comment:
This is going to be a breaking change so `As of mocha v7.0.0`

Reply:
Agreed.

Reply:
Okay :) I fixed it!